Fastfile

```diff
@@ -272,6 +273,44 @@ reflected in `Info.plist` during building."
 
   end
 
+  desc "Generates the push notifications certificates for the build configurations mapped to `:test` and `:qa`."
+  lane :generate_push_certificates_development do |options|
+    generate_push_certificates build_configuration: :test
+    generate_push_certificates build_configuration: :qa
+  end
+
+  desc "Generates the push notifications certificates for the build configurations mapped to `:appstore`."
+  lane :generate_push_certificates_appstore do |options|
+    generate_push_certificates build_configuration: :appstore
```

Comment:
why no `production` lane?

Reply:
We use `production` for external account.

I've had serious troubles with naming. I hear for suggestions.

I'm thinking of starting to use: `release_qa`, `release_internal_appstore`, `release_external_appstore`

Reply:
I think the names are okeay. But can't the one with access to the external account use these scripts to deploy? So to avoid bugs when archiving or other.

Reply:
Done in a branch later: https://github.com/Wolox/fastlane-mobile/pull/7/files